### ISSUE TITLE
Request for help about "arXiv:2211.09702v2"

### ISSUE THREAD

**Issue by 15738897318** [baturaysaglam/RIS-MISO-PDA-Deep-Reinforcement-Learning#2]
Dear Dr. Baturay Saglam,

I am a graduate student from the Harbin Institute of Technology, Shenzhen, China. My major is Information &Communication Engineering. My research interests include convex optimization in the reconfigurable intelligent surface-aided MIMO communication.

Second, I found your articles, such as one paper titled " Deep Reinforcement Learning Based Joint Downlink Beamforming 
and RIS Configuration in RIS-aided MU-MISO Systems 
Under Hardware Impairments and Imperfect CSI".

Next, I have the following questions:

For two different environment models: true environment with phase-dependent RIS amplitude and perfect CSI, and the mismatch
environment with ideal reflection assumption and imperfect CSI.

The proposed framework, which adopts a DRL method with SAC + β-Space Exploration in the mismatch cases,
outperforms the vanilla DRL agent (SAC) under mismatch, approaches the vanilla DRL agent (SAC) the golden
standard.

Problem 1:
Compare Figure.1 (d) in your paper with Figure.6 in the references. [1], I cannot understand the intention and reasons for your proposed two models.

Problem 2:
Which factor dominates the different combinations of these factors (i.e., phase-dependent RIS amplitude and perfect CSI, ideal reflection assumption and imperfect CSI)? 

In other words, the conventional approach is to fix one of the conditions, 
such as imperfect CSI, and conduct comparative testing (i.e., phase-dependent RIS amplitude and imperfect CSI, ideal reflection assumption and imperfect CSI).

It will do great help for me if you could provide me with some more detailed guidelines. 

Thank you for your kind consideration of this request.  I am looking forward to your reply.

Best wishes,

Meng Gao

**Comment by baturaysaglam** [comment]
Hi,

**Problem 1:** The [paper [1]](https://ieeexplore.ieee.org/document/9148961) in the references introduces the _practical phase dependent reflection amplitude_ model, considering the hardware impairments in the RIS. This is a realistic approach for the widely-studied RIS beamforming and phase-shift designs. The current ML- and DRL-based approaches should have considered this case and, therefore, remain unrealistic. Our study intends to devise the first DRL-based approach for this phase-dependent reflection amplitude model in RIS-aided MU-MISO systems to provide an alternative ML-based framework for the suboptimal iterative algorithms proposed in [1]. Our DRL-based approach performs superior without requiring extensive mathematical analysis, as done in [1].

**Problem 2:** Our simulation results show that phase-dependent RIS amplitude is the dominating factor. In other words, the DRL agent under imperfect CSI can still perform notably well without the phase-dependent RIS amplitude. However, when the phase-dependent RIS amplitude is considered, the performance drops while the DRL agent assumes perfect reflection. We solve this with our β-Space Exploration approach. 

Just to remind you, our paper is a workshop paper, which follows the strict page limit. Therefore, we couldn't simulate different combinations of the phase-dependent RIS amplitude model and imperfect CSI.

Please let me know if you have any other questions.
